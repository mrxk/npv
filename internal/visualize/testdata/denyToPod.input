---
apiVersion: networking.k8s.io/v1
kind: NetworkPolicy
metadata:
  name: denyToPod
  namespace: default
spec:
  podSelector:
    matchLabels:
      app: demo
  policyTypes:
    - Ingress
    - Egress
