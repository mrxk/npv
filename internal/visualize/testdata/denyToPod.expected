@startuml
left to right direction
frame Pods {
component "Name: denyToPod\lNamespace: default\lMatch Labels:\l    app: demo\l" as defaultappdemo {
    port "0-65535" as defaultappdemo_ALL_port
    portout " " as defaultappdemoportout
}
}
frame Ingress {
component "ALL" as _ALL_PEER_INGRESS__i {
    portout " " as _ALL_PEER_INGRESS_ingressportout
}
}
_ALL_PEER_INGRESS_ingressportout --down[#red]--> defaultappdemo_ALL_port
frame Egress {
component "ALL" as defaultappdemo_ALL__e {
    port "0-65535" as defaultappdemo_ALL_egressport
}
}
defaultappdemoportout --down[#red]--> defaultappdemo_ALL_egressport
@enduml
